test/ILLink.RoslynAnalyzer.Tests/TestChecker.cs

```diff
@@ -218,15 +282,15 @@ private bool TryValidateLogContainsAttribute (AttributeSyntax attribute, List<Di
 				}
 			}
 
-			missingDiagnosticMessage = $"Could not find text:\n{text}\nIn diagnostics:\n{string.Join (Environment.NewLine, DiagnosticMessages)}";
+			missingDiagnosticMessage = $"Could not find text:\n{text}\nIn diagnostics:\n{(string.Join (Environment.NewLine, _diagnostics))}";
 			return false;
 		}
 
-		private void ValidateLogDoesNotContainAttribute (AttributeSyntax attribute, List<Diagnostic> diagnosticMessages)
+		private void ValidateLogDoesNotContainAttribute (AttributeSyntax attribute, IReadOnlyList<Diagnostic> diagnosticMessages)
 		{
 			var arg = Assert.Single (LinkerTestBase.GetAttributeArguments (attribute));
 			var text = LinkerTestBase.GetStringFromExpression (arg.Value);
-			foreach (var diagnostic in DiagnosticMessages)
+			foreach (var diagnostic in _diagnostics)
```

Comment:
I dealt with this in another change - this should very likely be `diagnosticMessages` (the method parameter), otherwise the method parameter is not used. And I think it makes sense (looking at the callsite).

Reply:
I fixed it now.